packages/create-astro/src/index.ts

```diff
@@ -43,12 +44,23 @@ export async function main() {
 
 		// Steps which write to files need to go above git
 		git,
-		next,
 	];
 
 	for (const step of steps) {
 		await step(ctx);
 	}
+
+	// eslint-disable-next-line no-console
+	console.log('');
+
+	const labels = {
+		start: 'Initializing project...',
+		end: 'Project initialized!',
```

Comment:
These labels are intentionally mirrored so that the words don't jump around
```suggestion
		start: 'Project initializing...',
		end: 'Project initialized!',
```